model/utils_test.go

```diff
@@ -885,7 +885,7 @@ func TestIsValidHTTPURL(t *testing.T) {
 		},
 		{
 			"url with invalid scheme",
-			"htp://mattermost.com",
+			"http://mattermost.com",
```

Comment:
ok, this fix is wrong. But, this isn't a reasonable scheme. Use `example:`

https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml

`htp` could be registered.